app/Modules/ExpertPanel/Models/ExpertPanel.php

```diff
@@ -199,37 +196,6 @@ public function evidenceSummaries()
         return $this->hasMany(EvidenceSummary::class);
     }
 
-    /**
-     * @return \Illuminate\Database\Eloquent\Relations\HasOne
-     */
-    public function leadershipAttestation()
-    {
-        return $this->hasOne(\App\Modules\ExpertPanels\Models\LeadershipAttestation::class);
-    }
-
-    /**
-     * @return \Illuminate\Database\Eloquent\Relations\HasOne
-     */
-    public function biocuratorOnboardingAttestation()
```

Comment:
This does not appear to be referenced anywhere in the codebase and the table in the dump you recently sent me is empty.